assets/js/phoenix_live_view.js

```diff
@@ -1578,7 +1578,8 @@ export class View {
         params: this.connectParams(),
         session: this.getSession(),
         static: this.getStatic(),
-        flash: this.flash
+        flash: this.flash,
+        lv_version: __PACKAGE_VERSION__
```

Comment:
```suggestion
        vsn: this.parent ? undefined : PACKAGE_VSN
```

We should only send the version on the root LiveViews to avoid sending unnecessary data and checks. `undefined` values are automatically stripped form the JSON payload in JS land. We will also need to conditional check for the vsn param in the elixir side. Note I renamed `lv_version` to `vsn` as well. Thanks!